src/postman/__tests__/operation.test.ts

```diff
@@ -0,0 +1,89 @@
+import { HeaderDefinition, RequestBody } from 'postman-collection';
+import { transformPostmanCollectionOperation } from '../operation';
+
+describe('transformPostmanCollectionOperation()', () => {
+  describe('operation can be found', () => {
+    describe('description is set', () => {
+      it('returns operation with description', () => {
+        expect(
+          transformPostmanCollectionOperation({
+            document: {
+              item: [
+                {
+                  request: {
+                    method: 'get',
+                    url: '/path/:param?a=b',
+                    body: { mode: 'raw', raw: 'test' } as RequestBody,
+                    header: [{ key: 'header', value: 'a header' }] as HeaderDefinition,
```

Comment:
Hmm, I'll try to check out if we can get rid of this. @karol-maciaszek how did you generate this sample document? Did you use Postman?

Reply:
Initially, I used a tool from here https://github.com/postmanlabs/openapi-to-postman

But Postman app also generates it like that:
- `header` is supposed to be single HeaderDefinition while in reality, it is an array of such
- `body` is supposed to be RequestBody while in reality, it is a RequestBodyDefinition

Here is a screenshot of what has been generated by Postman app:
<img width="764" alt="Screen Shot 2020-02-05 at 12 50 14" src="https://user-images.githubusercontent.com/7136002/73839360-1b234c80-4816-11ea-8b18-e5783bda21c0.png">

I strongly believe those are just bugs in typings (e.g. the contributor saw "header" so assumed it has to be single one). I think when we finish the implementation we need to search for casts along our lines and create a fat PR to postman repo.

Reply:
Agreed, let's take note of that and we'll do such thing.